### ISSUE TITLE
analyze accessibility metadata

### ISSUE THREAD

**Issue by marisademeglio** [daisy/ace-core#35]
Right now, Ace summarizes what's present/missing regarding the accessibility metadata in the package document. There are two ways to improve upon this:

* include what's in the content documents (perhaps as a part of content analysis -- see this example of embedded video metadata: https://idpf.github.io/a11y-guidelines/content/meta/schema.org.html#meta010-ex02)
* suggest values for missing metadata

**Comment by murata0204** [comment]
Yoshimura-san and Ando-san considered which value of
accessibilityFeature should be added by programs.  Here is a rough
proposal.

## 1) Always assign these values

readingOrder
tableOfContents


## 2) Programs can determine these values

MathML
ttsMarkup

## 3) Programs can suggest these values

annotations
alternativeText
audioDescription
bookmarks
captions
describedMath
longDescription
displayTransformability
printPageNumbers
rubyAnnotations
structuralNavigation
synchronizedAudioText
timingControl
unlocked
transcript

## 4) Not sure

braille
ChemML
latex
highContrastAudio
highContrastDisplay
largePrint
taggedPDF
signLanguage
tactileGraphic
tactileObject

**Comment by mattgarrish** [comment]
You cannot always assume that table of contents and reading order apply. It still takes manual evaluation to check the quality. Incorrectly listed documents in the spine is not a feature, for example, nor would be a table of contents with one link to the body matter. There are other ways these could be done so poorly as to be useless for accessibility.

I would only suggest them, but, yes, they could always be suggested.

**Comment by rdeltour** [comment]
Thank you Murata-san, Yoshimura-san, and Ando-san! The list sound about right to me, with the added caveat from Matt.
